src/idl_parser.cpp

```diff
@@ -1566,6 +1590,12 @@ CheckedError Parser::ParseEnum(bool is_union, EnumDef **dest) {
       if (Is('=')) {
         NEXT();
         ev.value = StringToInt(attribute_.c_str());
+        // TODO: values between INT64_MAX and UINT64_MAX get silently truncated
```

Comment:
We still want these kinds of values to be representable, so the correct behavior is to store them bitwise in a signed number